.github/release-drafter.yml

```diff
@@ -0,0 +1,26 @@
+name-template: v$NEXT_PATCH_VERSION 🌈
+tag-template: v$NEXT_PATCH_VERSION
+categories:
+  - title: 🚀 Features
+    label: feature
+  - title: 🐛 Bug Fixes
+    label: fix
```

Comment:
This list would need to be adapted to the tags that we actually use: `enhancement`, `bug`, and so on. I would probably remove the icons - I don't think they add much.